sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/transaction/WalletTransactionEncoder.kt

```diff
@@ -157,7 +162,8 @@ internal class WalletTransactionEncoder(
                 twig("params exist! attempting to shield...")
                 rustBackend.shieldToAddress(
                     usk,
-                    memo
+                    memo,
+                    useZip317Fees
                 )
             } catch (t: Throwable) {
                 // TODO [#680]: if this error matches: Insufficient balance (have 0, need 1000 including fee)
```

Comment:
```suggestion
                // TODO [#680]: if this error matches: Insufficient balance (have 0, need * including fee)
```

Reply:
This part of the code is not altered by this PR anymore, and the Android SDK will stick with a fixed fee for the next release.